### ISSUE TITLE
Overhaul add transaction experience on desktop

### ISSUE THREAD

**Issue by Lem0naise** [Lem0naise/cashcat#249]
Should be smooth and snappy and keyboard focused, like entering a row in a spreadsheet, able to use keyboard shortcuts like tab and enter etc
